src/xattr.c

```diff
@@ -425,7 +425,7 @@ ssize_t scoutfs_getxattr(struct dentry *dentry, const char *name, void *buffer,
 
 	/* only need enough for caller's name and value sizes */
 	bytes = sizeof(struct scoutfs_xattr) + name_len + size;
-	xat = kmalloc(bytes, GFP_NOFS);
+	xat = kzalloc(bytes, GFP_NOFS);
```

Comment:
This xat buffer in getxattr (and in list and drop) is overwritten by item data.  It doesn't need to be zeroed.  These allocations include the full item so these can be 64k.